docs/release_notes.md

```diff
@@ -1,5 +1,175 @@
+  Version](usage/sat_bootprep.md#providing-a-schema-version).
+
+- Ability to look up images and recipes provided by products
+
+  For more information, see [Defining IMS
+  Images](usage/sat_bootprep.md#defining-ims-images).
+
+The schema of the `sat bootprep` input files was also changed to support these
+new features:
+
+- The base recipe or image used by an image in the input file should now be
+  specified under a `base` key instead of under an `ims` key. The old `ims`
+  key is deprecated.
+- To specify an image that depends on another image in the input file, the
+  dependent image should specify the dependency under `base.image_ref`.
+  You should no longer use the IMS name of the image on which it depends.
+- The image used by a session template should now be specified under
+  `image.ims.name`, `image.ims.id`, or `image.image_ref`. Specifying a string
+  value directly under the `image` key is deprecated.
+
+For more information on defining IMS images and BOS session templates in the
+`sat bootprep` input file, see [Defining IMS Images](usage/sat_bootprep.md#defining-ims-images)
+and [Defining BOS Session Templates](usage/sat_bootprep.md#defining-bos-session-templates).
+
+### Added Blade Swap Support to `sat swap`
+
+The `sat swap` command was updated to support swapping compute and UAN blades
+with `sat swap blade`. This functionality is described in the following processes
+of the [*Cray System Management Documentation*](https://cray-hpe.github.io/docs-csm):
+
+- **Adding a Liquid-cooled blade to a System Using SAT**
+- **Removing a Liquid-cooled blade from a System Using SAT**
+- **Replace a Compute Blade Using SAT**
+- **Swap a Compute Blade with a Different System Using SAT**
+
+### Support for BOS `v2`
+
+A new `v2` version of the Boot Orchestration Service (BOS) is available in CSM
+1.3.0. SAT has added support for BOS `v2`. This impacts the following commands
+that interact with BOS:
+
+- `sat bootprep`
+- `sat bootsys`
+- `sat status`
+
+By default, SAT uses BOS `v1`. However, you can choose the BOS version you want
+to use. For more information, see [Change the BOS Version](usage/change_bos_version.md).
+
+### Added BOS Fields to `sat status`
+
+When using BOS `v2`, `sat status` outputs additional fields. These fields show
+the most recent BOS session, session template, booted image, and boot status for
+each node. An additional `--bos-fields` option was added to limit the output of
+`sat status` to these fields. The fields are not displayed when using BOS `v1`.
+
+### Open Source Repositories
+
+This is the first release of SAT built from open source code repositories.
+As a result, build infrastructure was changed to use an external Jenkins instance,
+and artifacts are now published to an external Artifactory instance. These
+changes should not impact the functionality of the SAT product in any way.
+
+### Security
+
+#### CVE Mitigation
+
+The `paramiko` Python package version was updated from 2.9.2 to 2.10.1 to
```

Comment:
Another CVE item could be added here - 

> The `oauthlib` Python package version was updated from 3.2.0 to 3.2.1 to mitigate CVE-2022-36087.

https://github.com/Cray-HPE/sat/blob/release/3.19/CHANGELOG.md#security